config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-ipam
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+    - name: ANSIBLE_ENABLE_TASK_DEBUGGER
+      value: "True"
+  services:
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - ovn
+    - libvirt
+    - nova
+  deployStrategy:
+      deploy: true
+  baremetalSetTemplate:
+    deploymentSSHSecret: dataplane-ansible-ssh-private-key-secret
+    bmhLabelSelector:
+      app: openstack
+    ctlplaneInterface: enp1s0
+    dnsSearchDomains:
+      - osptest.openstack.org
+    cloudUserName: root
```

Comment:
Why're we changing to use root user here and in L36, we should use non-root user (cloud-admin) like before.

Reply:
by cloud admin you mean the cloud image user like centos/rhel right or a dedicated user like stack which has passwordless sudo right but is not root.

we want to use a non root user as root login for ssh should be disabled and we want
to make sure that any ansibel we are executing uses become properly which we can guarantee if we use root.

Reply:
cloud-admin the default user[1] for remote access, but it can be any non-root user with sudo rights.

https://github.com/openstack-k8s-operators/openstack-baremetal-operator/blob/main/api/v1beta1/openstackbaremetalset_types.go#L105-L107